app/models/Notification.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+import java.time.LocalDateTime
+
+import play.api.libs.json.Json
+
+case class Notification(dateTimeReceived: LocalDateTime, conversationId: String) extends Ordered[Notification] {
+
+  def compare(that: Notification): Int =
+    if (this.dateTimeReceived == that.dateTimeReceived) 0
```

Comment:
Could just subtract the two dates?

Reply:
`compare` is just to be able to sort dates. I's not related with substracting etc.

Reply:
Every time you do an `if` you need two tests, so to work out if one date is bigger than another this method essentially needs 3 unit tests.

In fact all you have to do is `this.dateTimeReceived.compareTo(that.dateTimeReceived)`
If you dig into the implementation of that method though all it does is subtract one date from the other

Reply:
This has been addressed.